snap/snapcraft.yaml

```diff
@@ -60,7 +58,6 @@ parts:
       install -DT ./helper-go $SNAPCRAFT_PART_INSTALL/bin/helper-go
 
   device-rest:
-    after: [metadata]
     source: .
     plugin: make
     build-packages: [git, zip, pkg-config]
```

Comment:
These were probably needed for libzmq. No longer needed.

Reply:
It appears that the git package is required, as an error is present without it:
```
/bin/bash: line 35: git: command not found
```

Reply:
Strange. I know that Go compiler needs it. Maybe it isn't included by the Go plugin on the core20 base.